public/classroom_poc.html

```
@@ -0,0 +1,839 @@
+                               class="w-full pl-10 pr-4 py-2.5 bg-gray-800 border border-gray-700 rounded-lg text-sm text-gray-100 placeholder-gray-500 focus:outline-none focus:ring-2 focus:ring-teal-500/50 focus:border-teal-500 transition" />
+                    </div>
+                </div>
+                <div>
+                    <label for="lobbyDisplayName" class="block text-xs font-semibold text-gray-400 uppercase tracking-wider mb-1.5">Display Name</label>
+                    <div class="relative">
+                        <i data-lucide="user" class="absolute left-3 top-1/2 -translate-y-1/2 w-4 h-4 text-gray-500"></i>
+                        <input id="lobbyDisplayName" type="text" placeholder="Your name"
+                               class="w-full pl-10 pr-4 py-2.5 bg-gray-800 border border-gray-700 rounded-lg text-sm text-gray-100 placeholder-gray-500 focus:outline-none focus:ring-2 focus:ring-teal-500/50 focus:border-teal-500 transition" />
+                    </div>
+                </div>
+                <button id="btnJoin"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)